documentation/book/proc-deploying-kafka-connect-openshift.adoc

```diff
@@ -3,22 +3,23 @@
 // assembly-kafka-connect.adoc
 
 [id='deploying-kafka-connect-openshift-{context}']
-= Deploying Kafka Connect to {OpenShiftName}
+= Deploying Kafka Connect to your {OpenShiftName} cluster
 
-On {OpenShiftName}, Kafka Connect is provided in the form of a template. It can be deployed from the template using the command-line or through the {OpenShiftName} console.
+To allow you to build connectors for your Kafka cluster, you can deploy a Kafka Connect cluster to your {OpenShiftName} cluster by using the Cluster Operator. Kafka Connect is provided as an {OpenShiftName} template that you can deploy from the command line or the {OpenShiftName} console.
 
 .Prerequisites
 
-* Before deploying Kafka Connect, the Cluster Operator must be deployed.
+* xref:deploying-cluster-operator-openshift-{context}[Deploying the Cluster Operator to {OpenShiftName}]
```

Comment:
I believe you want to replace {context} here with whatever the context was on the page you're linking to